tempesta_fw/t/unit/test_tfw_str.c

```diff
@@ -1178,6 +1178,292 @@ TEST(tfw_str_collect_cmp, collect_chunks)
 	EXPECT_EQ(TFW_STR_CHUNKN(&out), 5);
 }
 
+TEST(tfw_str_add_compound, plain)
+{
+	TfwStr *s;
+	TfwStr s1 = {
+		.len = 16,
+		.ptr = "abcdefghijklmnop"
+	};
+	TfwStr s2 = {
+		.len = 10,
+		.ptr = "qrstuvwxyz"
+	};
+
+	s = tfw_str_add_compound(str_pool, &s1);
+	*s = s2;
```

Comment:
This is the unit test. `tfw_str_add_compound()` can return NULL value. Crash will happen and we definitely will spot the bug. But it would be nice to give exact warning directly. Please add `EXPECT_NON_NULL(s)` before use it.